docs/source/usage/python.rst

```diff
@@ -611,6 +611,22 @@ This module provides elements for the accelerator lattice.
    :param unit: specification of units (``"dimensionless"`` in units of the magnetic rigidity of the reference particle or ``"T-m"``)
    :param name: an optional name for the element
 
+.. py::class:: impactx.elements.LinearMap(R, dx=0, dy=0, rotation=0, name=None)
+
+   A custom, linear transport matrix.
+
+   The matrix elements :math:`R(i,j)` are indexed beginning with 1, so that :math:`i,j=1,2,3,4,5,6`.
+   The transport matrix :math:`R` is defaulted to the identity matrix, so only matrix entries that differ from that need to be specified.
```

Comment:
No true for Python, where we set the whole matrix `R` via assignment. Our examples show how to start from an identity matrix, but its up to the user if they follow that.
```suggestion
```